shotgun_api3/shotgun.py

```diff
@@ -945,23 +945,39 @@ def followers(self, entity):
         
         return self._call_rpc('followers', params)
 
-    def schema_entity_read(self):
+    def schema_entity_read(self, project_entity=None):
```

Comment:
Hey there! It would be great if we could have a `:param: style docstring for this new parameter - just for consistency with other methods! Thanks!